WcaOnRails/config/puma.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+# Puma can serve each request in a thread from an internal thread pool.
```

Comment:
I'm open to getting rid of unicorn, but as @larspetrus said, I see that happening sometime after switching to Rails 5 (or before, but not simultaneously =))